bound/kt/src/main/kotlin/web5/sdk/crypto/keys/Jwk.kt

```diff
@@ -1,10 +1,44 @@
 package web5.sdk.crypto.keys
 
+import web5.sdk.rust.Jwk as RustCoreJwk
 import web5.sdk.rust.JwkData as RustCoreJwkData
 
 /**
  * Partial representation of a [JSON Web Key as per RFC7517](https://tools.ietf.org/html/rfc7517).
  * Note that this is a subset of the spec.
  */
+data class Jwk (
```

Comment:
by the way, why don't we have a `kid` field? we have them in the key for the test vectors - https://github.com/TBD54566975/web5-spec/blob/main/test-vectors/crypto_ed25519/sign.json

Reply:
Because we haven't have an immediate application use case for it

Created #297 feel free to add it!